DOCUMENTATION.md

```diff
@@ -1,17 +1,69 @@
 # Documentation
 
-Protogenie is a tool developed around configuration files written in XML. This design choice was made
-to allow for validation and control of configuration (this way, if your configuration file is valid but
-it does not do what it should do, we know it's a bug !).
+The purpose of _Protogenie_ is to prepare data to train a lemmatiser and/or POS tagger.
+All the training data is split into three different files:
+1. A train set
+2. An development set
+3. A testing (or evaluation) set
+
+<span style="color:red; font-weight:bold">_Protogenie_ expects CSV- or TSV-like resources.</span>
+
+After running _Protogenie_, the result should be the following
+
+```txt
+FOLDER
+|-mydata.tsv
+|-OUTPUT
+    |-DEV
+        |-mydata.tsv
+    |-TEST
+        |-mydata.tsv
+    |-TRAIN
+        |-mydata.tsv
+```
 
-The scheme is available inside the package, and with each version by simply typing `protogenie get-scheme [DESTINATION]`.
- You can also find the latest version here: 
- [https://hipster-philology.github.io/protogenie/protogenie/schema.rng](https://hipster-philology.github.io/protogenie/protogenie/schema.rng)
+Using the same exmaple, the repartition of the data between the three files is done directly with the command line:
+
+```bash
+$ protogenie build mydata.tsv -d 0.1 -e 0.1
+```
+
+The `-d` (for `development`) and the `-e` (for `evaluation` or testing) flags are followed by a decimal number representing the percentage taken in the original dataset (`0.1`=`10%`). The final breakdown would therefore be the following:
+
+1. 10% for the dev set
+2. 10% for the eval/test set
+3. 80% for the train set
+
+<span style="color:red; font-weight:bold">_Protogenie_ does not create test sets directly from tokens, but from groups of tokens (_e.g._ sentences). If the dataset is small, the amount of tokens might not be equal to the expected percentages.</span>
```

Comment:
On the other hand, on rather larger corpora (>200/300k tokens), this should be ok.

Reply:
corrigé